string guid
int32 framenumber_offset
string pickled_missing_data
---
